### ISSUE TITLE
Pin Description not populated from img alt/title attribute, for some images

### ISSUE THREAD

**Issue by ghodgson** [pinterest/widgets#46]
Hello,
I'm using the Firefox extension "Pin it Button" v 1.35 on Firefox 34.0.5.

I've found that when pinning, the pre-populated "Description" value varies depending on the actions that I use to select the image to pin.

Hovering over the image and selecting the "Pin it" button  causes the Description to be loaded from the alt/title attribute of the image.

However, clicking the "Pin It" icon from Firefox's toolbar displays all pinnable images on the page.  If I select the same image as before, in this case the first paragraph of text on the page is used as the description.  I would expect in this case the img/alt text of the image would still be used.

How can I ensure that the pre-populated description uses the alt or title tag of the image?

This only happens for a few of our images.  I've created a test page which includes one such affected image:  http://thethriftyginger.com/pinterest-debug-page/

A quick video demo of this behavior can be seen here:  http://youtu.be/QKbUmVb-hE4

**Comment by kentbrew** [comment]
We're picking up an og:image from the header of this page and using its og:description for the default. To fix this, please change the og:description from "Visit the post for more" to something more useful.

To exactly control the default description of an image, use data-pin-description="your preferred description here" on the IMG tag. Further details here:

https://business.pinterest.com/en/blog/pin-it-button-technical-tune-5-tips-make-sharing-your-site-better

**Comment by ghodgson** [comment]
Thanks for the super-fast response Kent!

I've set up a second test page with two images on it:  http://thethriftyginger.com/pinterest-debug-page-2/  I've set the titles explicitly using the data-pin-description attribute as you described, however I still see inconsistent behavior when pinning one image or the other.

When I click the "Pin It" button in the Firefox toolbar and select the black image that reads "One", the pre-filled description displays the value from the image's data-pin-description attribute ("Image One - Alt Text").  All is good here!

However, when I click the "Pin It" button and select the black image that reads "Two", the pre-filled description displays the page's og:description meta value instead of the image's data-pin-description attribute ("Image Two - Alt Text").

Is it supported behavior to have multiple "data-pin-description"s on the same page like this?
